pull_request_template.md

```diff
@@ -1,5 +1,6 @@
 ### What issue is this solving?
-Please post a link to the issue (on the front end repo) that this PR is resolving.
+<!-- replace ### with the issue number. This will ensure the issue is automatically closed when the PR is merged. -->
+Closes https://github.com/codeforpdx/dwellingly-app/issues/###
```

Comment:
```suggestion
<!-- replace ### with the issue number. This will ensure the issue in the FE repo is automatically closed when the PR is merged. -->
Closes codeforpdx/dwellingly-app/issues/###
```

Use relative link